source/common/json/json_streamer.h

```diff
@@ -38,6 +38,26 @@ namespace Json {
 #define ASSERT_LEVELS_EMPTY ASSERT(this->levels_.empty())
 #endif
 
+class Constants {
+public:
+  // Constants for common JSON values.
+  static constexpr absl::string_view True = "true";
```

Comment:
I still feel like we are better off without these, especially without the other ones. Reminds me of code like:

```
   x = 5;   // sets x to five
```

Reply:
I will argue for that. This is more like define the constant header name like:

```
constexpr absl::string_view HeaderName = "XXXXXX";
```

This allow all positions that need to use the value needn't to spell the literal string and avoid potential spelling error. I revert the delimiter part because I found I won't use these delimiter outside of `Streamer`. But we still use to use these values (true, false, null) in the `JsonFormatter` for some historical reasons.